### ISSUE TITLE
Static network setup

### ISSUE THREAD

**Issue by jpschewe** [jpschewe/fll-sw#541]
Since the web based judging application is tied to the IP of the server, if the server changes IP addresses due to a router outage or network change, then the scores cannot be uploaded. 

Possible ideas:
- multicast DNS (need something on the routers for this so that the tablets and windows understand)
- DHCP reservations (need custom router firmware)
- static IP addresses on all machines and then make the routers all be in the same network space

**Comment by eengstrom** [comment]
Yeah!!!!  I've wanted this for years - In fact, I think I submitted a bug on it in the old system.

Regardless, another idea is if the routers support (local) name resolution, have the system/software (temporarily) change the hostname to "fll-server" or equivalent.  

At the same time, let's get off the non-standard port.  Just put it on port 80 an be done with it.

**Comment by jpschewe** [comment]
The problem with port 80 is that you need to be root or admin to use it.
That is not very user friendly. You can out in a separate ticket for this
and we will discuss there.

Changing the hostname has similar problems as well.

At this point I want to handle router reboot.

Sent from my mobile device


On Jan 26, 2017 08:50, "Eric Engstrom" <notifications@github.com> wrote:

Yeah!!!! I've wanted this for years - In fact, I think I submitted a bug on
it in the old system.

Regardless, another idea is if the routers support (local) name resolution,
have the system/software (temporarily) change the hostname to "fll-server"
or equivalent.

At the same time, let's get off the non-standard port. Just put it on port
80 an be done with it.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/jpschewe/fll-sw/issues/541#issuecomment-275405020>, or mute
the thread

**Comment by jpschewe** [comment]
See #527 and #277 for ideas on discovery.

**Comment by jpschewe** [comment]
See #323 about how to handle a different web port.

**Comment by jpschewe** [comment]
I've configured FLL-11 as the server for set 1 and FLL-12 as the server for set 2. This has been updated in the inventory spreadsheet. Each server has been paired with a router and has been given the .2 address from the router. The tablets for each set have visited the server address and have it bookmarked.

As we get the rest of the computers for the 2 tournament sets defined we should put bookmarks on those as well.
